tasks/libs/common/utils.py

```diff
@@ -228,10 +228,25 @@ def get_build_flags(
     elif os.environ.get("NO_GO_OPT"):
         gcflags = "-N -l"
 
-    # On macOS work around https://github.com/golang/go/issues/38824
-    # as done in https://go-review.googlesource.com/c/go/+/372798
     if sys.platform == "darwin":
-        extldflags += "-Wl,-bind_at_load "
+        # On macOS work around https://github.com/golang/go/issues/38824
+        # as done in https://go-review.googlesource.com/c/go/+/372798
+        extldflags += "-Wl,-bind_at_load,-no_warn_duplicate_libraries"
```

Comment:
This looks like it's adding it by default and then adding it a second time in the conditional?

Reply:
Oups forgot to remove this, thanks ! Fixing it !